src/akar/util.clj

```diff
@@ -0,0 +1,48 @@
+(ns akar.util
+  (:require [clojure.walk :as cw]))
+
+
+(declare ^{:doc "All the occurances of this symbol will be replaced by
+recursive function call when used inside body passed to
+defn-trampolined.  DO NOT use this symbol outside of
+defn-trampolined."} trampolined-recur)
```

Comment:
And also the symbol. It may be easier to read that way.

Reply:
Removing the var from this PR